### PR TITLE
Switch to using spp-logger for logging

### PR DESCRIPTION
**Pull request by arthompson**

### CODE REVIEW

**Review by glanvl — APPROVED** [review]

File: strata_period_method.py [hunk 1]
```diff
@@ -27,11 +27,13 @@ def handle_error(self, e, data, **kwargs):
         raise ValueError(f"Error validating runtime params: {e}")
 
     data = fields.Str(required=True)
+    environment = fields.Str(required=True)
     current_period = fields.Str(required=True)
```
Comment (Piwington) [inline strata_period_method.py]: C before D.

File: strata_period_method.py [hunk 2]
```diff
@@ -27,11 +27,13 @@ def handle_error(self, e, data, **kwargs):
...
     period_column = fields.Str(required=True)
     reference = fields.Str(required=True)
     region_column = fields.Str(required=True)
     segmentation = fields.Str(required=True)
+    survey = fields.Str(required=True)
     survey_column = fields.Str(required=True)
     bpm_queue_url = fields.Str(required=True)
```
Comment (Piwington) [inline strata_period_method.py]: B higher.

File: strata_period_method.py [hunk 3]
```diff
@@ -61,24 +60,37 @@ def lambda_handler(event, context):
...
-
         # Environment Variables
         strata_column = environment_variables["strata_column"]
         value_column = environment_variables["value_column"]
 
         # Runtime Variables
         bpm_queue_url = runtime_variables["bpm_queue_url"]
         data = runtime_variables["data"]
+        environment = runtime_variables['environment']
         current_period = runtime_variables["current_period"]
```
Comment (Piwington) [inline strata_period_method.py]: C before D.

File: strata_period_wrangler.py [hunk 4]
```diff
@@ -36,8 +36,10 @@ def handle_error(self, e, data, **kwargs):
     distinct_values = fields.List(fields.String, required=True)
     sns_topic_arn = fields.Str(required=True)
     survey_column = fields.Str(required=True)
+    survey = fields.Str(required=True)
     bpm_queue_url = fields.Str(required=True)
     total_steps = fields.Str(required=True)
+    environment = fields.Str(Required=True)
```
Comment (Piwington) [inline strata_period_wrangler.py]: B and E should be higher up.

File: strata_period_wrangler.py [hunk 5]
```diff
@@ -89,10 +86,27 @@ def lambda_handler(event, context):
         out_file_name = runtime_variables["out_file_name"]
         region_column = runtime_variables["distinct_values"][0]
         sns_topic_arn = runtime_variables["sns_topic_arn"]
+        survey = runtime_variables["survey"]
         survey_column = runtime_variables["survey_column"]
         total_steps = runtime_variables["total_steps"]
+        environment = runtime_variables['environment']
```
Comment (Piwington) [inline strata_period_wrangler.py]: E should be higher.

File: strata_period_wrangler.py [hunk 6]
```diff
@@ -109,11 +123,13 @@ def lambda_handler(event, context):
...
                 "period_column": period_column,
                 "segmentation": segmentation,
+                "survey": survey,
                 "survey_column": survey_column,
                 "reference": reference,
                 "region_column": region_column,
                 "run_id": run_id,
-                "bpm_queue_url": bpm_queue_url
+                "bpm_queue_url": bpm_queue_url,
+                "environment": environment
```
Comment (Piwington) [inline strata_period_wrangler.py]: B and E higher.

File: tests/test_strata.py [hunk 7]
```diff
@@ -48,6 +51,7 @@
             "sns_topic_arn": "fake_sns_arn",
             "distinct_values": ["region"],
             "bpm_queue_url": "fake_queue_url",
+            "environment": "sandbox",
```
Comment (Piwington) [inline tests/test_strata.py]: This whole block is messy.

**Review by Piwington — CHANGES_REQUESTED** [review]
Code is looking good but again Alphabetical issues.

File: strata_period_wrangler.py [hunk 8]
```diff
@@ -30,16 +30,16 @@ def handle_error(self, e, data, **kwargs):
...
+    bpm_queue_url = fields.Str(required=True)
+    distinct_values = fields.List(fields.String, required=True)
+    environment = fields.Str(Required=True)
     in_file_name = fields.Str(required=True)
     out_file_name = fields.Str(required=True)
-    distinct_values = fields.List(fields.String, required=True)
+    period = fields.Str(required=True)
     sns_topic_arn = fields.Str(required=True)
     survey_column = fields.Str(required=True)
     survey = fields.Str(required=True)
```
Comment (Piwington) [inline strata_period_wrangler.py:40]: I think as survey is shorter it should technically come before survey_column

File: tests/test_strata.py [hunk 9]
```diff
@@ -16,42 +16,42 @@
...
+            "bpm_queue_url": "fake_queue_url",
+            "distinct_values": ["region"],
+            "environment": "sandbox",
             "in_file_name": "test_wrangler_input",
             "out_file_name": "test_wrangler_output.json",
             "period": "201809",
+            "run_id": "bob",
+            "survey": "BMI_SG",
+            "survey_column": "survey",
             "sns_topic_arn": "fake_sns_arn",
```
Comment (Piwington) [inline tests/test_strata.py]: sn before su

**Review by Piwington — CHANGES_REQUESTED** [review]

**Review by Piwington — APPROVED** [review]

**Pull request merged**